kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/default-language-form/index.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+
+  <onboarding-form :header="$tr('languageFormHeader')" :submit-text="submitText" @submit="setLanguage">
+    <label
+      :class="[
+        'default-language-form-selected',
+        'default-language-form-items',
+        (isMobile ? 'mobile' : '')
+      ]">
+      <span :class="['default-language-form-selected-label', (isMobile ? 'mobile' : '')]">
+        {{ $tr('selectedLanguageLabel') }}
+      </span>
+      <span> {{ selectedLanguage }} </span>
```

Comment:
Spaces

Reply:
Not sure what you mean here. You want me to get rid of the spaces between the curly braces and the tags?